partner_contact_job_position/readme/USAGE.rst

```diff
@@ -0,0 +1,4 @@
+To use this module, you need to:
+
+* Go to any partner's form.
+* Use the new *Categorized job position* field.
```

Comment:
Please, add a few lines describing the option to add/see/edit Job positions in Contacts --> Configuration --> Job Positions

Reply:
I add how to config `job position` in configuration section. Is this same above text ?
![Selection_400](https://user-images.githubusercontent.com/24691983/96372530-6fe2d980-1191-11eb-8e2c-8dc2a83e6954.png)

Reply:
Ok, sorry I didn't notice there is in the configuration file